projects/storefrontapp-e2e-cypress/cypress/integration/regression/cpq-configuration.e2e-spec.ts

```diff
@@ -1,37 +1,63 @@
 import * as configuration from '../../helpers/product-configuration';
 import * as productSearch from '../../helpers/product-search';
 
-const powertoolsShop = 'powertools-spa';
+const POWERTOOLS = 'powertools-spa';
 const email = 'cpq03@sap.com';
 const password = 'welcome';
 const cpqUser = 'cpq03';
-const testProduct = 'CONF_CAMERA_BUNDLE';
-const testProductCoffeeMachine = 'CONF_COFFEEMACHINE_3000';
 
 // UI types
-const radioGroup = 'radioGroup';
-const checkBoxList = 'checkBoxList';
-
-// Attributes
-const ATTR_COFFEE_MACHINE_CUPS_DAY = '2931'; // COFFEE_MACHINE_CUPS_DAY
-const ATTR_COFFEE_MACHINE_STARB_MODE = '2933'; // COFFEE_MACHINE_STARB_MODE
-
-// Attribute values
-const COFFEE_MACHINE_CUPS_DAY_300_500 = '8841'; // 300-500 CUPS
-const COFFEE_MACHINE_CUPS_DAY_500_1000 = '8842'; //  500-1000 CUPS
-
-const STARB_MODE = '8845'; // STARB_MODE
-
-// List of groups
-const MAIN_COMPONENTS = 'Main Components';
-const ACCESSORIES = 'Accessories';
-const INSURANCE_AND_WARRANTY = 'Insurance and Warranty';
-
-// List of attributes
-const attributeHeaders = {
-  mainComponents: ['Camera Body', 'Memory Card', 'Lenses'],
-  accessories: ['Tripod', 'Bag', 'Special Accessories'],
-  insuranceAndWarranty: [
+const RB = 'radioGroup';
+const CBL = 'checkBoxList';
```

Comment:
According to my point of view, the constants names are **too short**. 
We cann't expect from everyone that he or she knows CPQ product by heart.

Reply:
These 2 terms are not related to CPQ? RB = RadioButton. Ok maybe RG= radioGroup would be better?
What would be a suitable contestant name in your eyes?

Reply:
Maybe, the names should reflect the UI names in a certain way in order to find them better in the UI